src/Components/Sections/SectionForm.js

```diff
@@ -0,0 +1,68 @@
+import React, { useRef, useState } from "react";
+import { Link } from "react-router-dom";
+import { Form, Button } from "react-bootstrap";
+import ReactQuill from "react-quill";
+import "./SectionForm.css";
+
+export default function SectionForm(props) {
+  const [sectionFields, setSectionFields] = useState({
+    title: "",
+    html: "",
+  });
+  const quillEl = useRef(null)
+
+  const handleCancel = (event) => {
+    event.preventDefault();
+    props.onCancel();
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    props.onSubmit({
+      ...sectionFields,
+      text: quillEl.current.getEditor().getText(),
```

Comment:
This allows us to get the text from the editor without any markup inside it. This should make the word counts more reliable.